pabi_budget_drilldown_report/xlsx_template/xlsx_template_wizard.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+      # Budget Drilldown
+      <!-- Budget Drilldown Unit Based -->
+      <act_window id="action_budget_drilldown_unit_export_xlsx_template"
+          name="Export Template (.xlsx)"
+          res_model="export.xlsx.template"
+          src_model="budget.drilldown.report"
+          view_mode="form"
+          target="new"
+          context="{'lang': 'th_TH'}"/>
```

Comment:
Add context, to ensure that it will be selected as default.
'template_fname': 'budget_overview_report.xlsx',